targets/TARGET_STM/sleep.c

```diff
@@ -84,6 +85,10 @@ void hal_deepsleep(void)
     // After wake-up from STOP reconfigure the PLL
     SetSysClock();
 
+    TIM_HandleTypeDef TimMasterHandle;
+    TimMasterHandle.Instance = TIM_MST;
+    __HAL_TIM_SET_COUNTER(&TimMasterHandle, EnterTimeUS);
```

Comment:
Thanks. For future reference, this type of details is often worth having in the commit message, for a reader (I just guessed this without looking at the reference manual ).